src/index.tsx

```diff
@@ -1,12 +1,12 @@
-import React, { StrictMode } from 'react';
+import React, { Fragment } from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import GlobalStyle from './ui/global/GlobalStyle';
 
 ReactDOM.render(
-  <StrictMode>
+  <Fragment>
```

Comment:
StrictMode removed as MUi's Drawer component doesn't comply with that mode. It's a known issue and should be replaced back to StrictMode when MU fixes it